golem-cluster.yaml

```
@@ -27,10 +27,19 @@ provider:
     node_config:
       # Parameters for golem demands (same for head and workers)
       demand:
-        # if not provided, image_tag will be autodetected based on currently used python and ray versions
-        # check available versions at https://registry.golem.network/explore/golem/ray-on-golem
+        # Check available versions at https://registry.golem.network/explore/golem/ray-on-golem
         image_tag: "golem/ray-on-golem:0.3.0-py3.10.13-ray2.7.1"
+
+        # List of urls which will be added to the Computation Manifest
+        # Requires protocol to be defined in all URLs
+        # If not provided demand will not use Computation Manifest
+        outbound_urls: ["https://pypi.dev.golem.network"]
+
+        # Defaults to `["vpn", "inet"]`
+        # If outbound_urls is specified `"manifest-support"` will be added
         capabilities: ["vpn", "inet", "manifest-support"]
```

```suggestion
        capabilities: ["vpn", "inet"]
```
let's remove the `manifest-support` - it's implied and keeping it might be misleading, hm?

---

or even better - we could remove capabilities it from here completely - it is not mandatory and we don't expect people edit it anytime soon, right?

---

I thought we wanted to showcase as many possibilities as possible in this yaml